plugins/plugin-bash-like/src/lib/util/ascii-key-value-to-table.ts

```diff
@@ -44,7 +44,7 @@ export default (str: string): Pair[] | void => {
     do {
       const value = { name: 'value', onclick: false, value: match[2], css: 'normal-wrap break-all' }
       matches.push({ type: 'key-value', name: match[1], attributes: [ value ] })
-    } while (match = kvPattern.exec(str))
+    } while (match === kvPattern.exec(str))
```

Comment:
here, the conditional assignment is necessary, so the fix will take more than this, i think

Reply:
I have disabled it for now then!